NorthstarDLL/mods/modmanager.cpp

```diff
@@ -398,10 +398,44 @@ void ModManager::LoadMods()
 	std::filesystem::directory_iterator classicModsDir = fs::directory_iterator(GetModFolderPath());
 	std::filesystem::directory_iterator remoteModsDir = fs::directory_iterator(GetRemoteModFolderPath());
 
+	this->m_invalidMods.clear();
+
 	for (std::filesystem::directory_iterator modIterator : {classicModsDir, remoteModsDir})
 		for (fs::directory_entry dir : modIterator)
+		{
 			if (fs::exists(dir.path() / "mod.json"))
 				modDirs.push_back(dir.path());
+			else if (fs::is_directory(dir.path()))
+			{
+				std::string filename = dir.path().filename().generic_string().c_str();
+				// Don't display an error for hidden directories
+				if (filename.at(0) == '.')
```

Comment:
Technically speaking you can have non-hidden directories that start with `.`

Example would be the `.github` folder

Not rly sure that it's a big deal in any way, if someone makes a mod/package that starts with `.` then thats their problem

Reply:
Yeah, a good example is indeed people directly cloning `NorthstarMods` into their game folder (just like https://github.com/R2Northstar/NorthstarLauncher/pull/405#issuecomment-1412805019); I think directories starting with a `.` are configuration-related folders, and thus mods shouldn't be named that way.